ui_tests/src/test/resources/com/saucedemo/account/signInNegative.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Sign In with Invalid Data
+
+  @signInNegative
+  Scenario: Invalid username sign in
+    Given I open start page
+    When I set 'muxxa_26' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+
+  @signInNegative
+  Scenario: Sign in with invalid password
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set 'thisIsSomeInvalidPassword' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+
+    @signInNegative
+    Scenario: Invalid password and username
+      Given I open start page
+      When I set 'shfsdghd' as username on Sign in page
+      And I set 'thisIsSomeInvalidPassword' as password on Sign in page
+      And I click 'Sign In' button on Sign in page expecting failure
+      Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+
+     @signInNegative
+     Scenario Outline: Invalid password or username
+       Given I open start page
+       When I set '<username>' as username on Sign in page
+       And I set '<password>' as password on Sign in page
+       And I click 'Sign In' button on Sign in page expecting failure
+       Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+
+       Examples:
+       |username   |password |
+       | standart_user|  ''  |
+       |  '' |   secret_sauce   |
+       | locked_out_user  |  secret_sauce    |
```

Comment:
@muxxa26 
1. Use Code > Reformat Code in order to automatically arrange the columns and fix obvious style issues:
![image](https://user-images.githubusercontent.com/3727228/78406967-c9a6fc00-75d1-11ea-9f25-9295af8d6321.png)
2. Please read through the documentation on how to use Scenario Outline:
https://cucumber.io/docs/gherkin/reference/#scenario-outline
And fix this test
The scenario outline is like a template of scenario that is executed with different sets of data.
The data for the scenario outline that is used as entry data for the scenario outline execution is stored as Examples: table in a form of a table with columns and rows. The very first row is the name of a column that is being used in scenario outline for linking between step and data in examples table. E.g. column name Username is referenced from I set '< username>' as username on Sign in page step as < username>.
As of now this scenario outline verifies that for all the combination of username and password you always have the very same error: Epic sadface: Username and password do not match any user in this service
But it should be verifying that for EACH combination of username and password you get the error message specific to this combination (the third column)
3. Setting nothing to a field (or empty values) was covered in the last class. In case of Examples table you should leave the cell empty (instead of the empty quotes)